package main

func get_exit_status(err error) int {
	return 1
}
